dist/fabric.js

```diff
@@ -10341,6 +10351,76 @@ fabric.BaseBrush = fabric.util.createClass(/** @lends fabric.BaseBrush.prototype
       return newPoints;
     },
 
+    /**
+     * collapse path by removing inner colinear points
+     */
+    simplifyPath: function(fullPath) {
+    // we start with [0], the M of the path
+    var newPath = [fullPath[0]] 
```

Comment:
Just me of is the indentation off here on the whole function?

Reply:
oops, you are right. Fixed.